packages/frontend/src/utils/risks/color.ts

```diff
@@ -1,23 +1,25 @@
-import { Sentiment } from './types'
+import { Sentiment } from '@l2beat/config'
 
-export function sentimentToFillColor(sentiment?: Sentiment): string {
+export function sentimentToFillColor(sentiment: Sentiment): string {
   switch (sentiment) {
     case 'bad':
       return 'fill-orange-600'
     case 'warning':
       return 'fill-yellow-200'
-    case undefined:
+    case 'good':
```

Comment:
so 'good' will be green and 'neutral' will be gray, right? what's the difference between 'neutral' and 'upcoming'?

Reply:
Yes, that's it. Upcoming risks are neutral because they do not exist yet. Theres also `Destination` "risk" in bridges that should be consider grey if any (imo should not be part of risks, but this is part of refining bridges page that we'll have to do some time in the future). There will also be 'In review' risk which will be different than 'neutral'. <img width="673" alt="Screenshot 2023-07-03 at 09 08 14" src="https://github.com/l2beat/l2beat/assets/38423054/62c12350-af71-42b0-8427-3810f98a5493">